src/ctapipe/image/cleaning.py

```diff
@@ -112,6 +112,41 @@ def tailcuts_clean(
         )
 
 
+def bright_cleaning(image, threshold=267, fraction=0.03):
```

Comment:
this is a weirdly precise number for a default argument

Reply:
for now i just used the default values from lstchain